### PR TITLE
Feature/palindrome

### PR DESCRIPTION
**Pull request by AndSak**

### CODE REVIEW

File: Palindrome/script.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  let halfLength = Math.floor(arr.length/2); 
+
+  for (let i = 0; i < halfLength; i++) {
+
+    if (arr[i] == arr[arr.length - 1 - i]) {
+      count++;
+    }
+  }
+
+  if (count == halfLength) return true
```
Comment (AndrZakhara) [inline Palindrome/script.js:15]: add curly braces so the code will be more readable and understandable
if() {
    ....
} else {
    ....
}

You can use: if() return value  - if you have one row only

File: Palindrome/script.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+function palindrome(str) {
+
+  let count = 0;
+  let arr = str.toLowerCase().split('').filter(char => /[a-z]/.test(char));
```
Comment (AndrZakhara) [inline Palindrome/script.js:4]: add numbers to regexp this is not a bug. read the assignment carefully pls
--
task:
...
Note: You'll need to remove all non-alphanumeric characters (punctuation, spaces and symbols) and turn everything into the same case (lower or upper case) in order to check for palindromes.

We'll pass strings with varying formats, such as racecar, RaceCar, and race CAR among others.

We'll also pass strings with special symbols, such as 2A3*3a2, 2A3 3a2, and 2_A3*3#A2.

Comment (AndSak) [reply Palindrome/script.js:4]: Oh  Zounds, I get it now, the numbers participate in this party too. Everything will be corrected

File: Palindrome/script.js (hunk 1 above)
Comment (AndSak) [reply Palindrome/script.js:15]: I now it , but sometimes i so do it , as the code is very short.
ok, I`ll improve

Comment (AndSak) [reply Palindrome/script.js]: know😁

**Review by AndrZakhara — APPROVED** [review]

**Review by AndrZakhara — APPROVED** [review]

**Pull request merged**